Cargo.toml

```diff
@@ -0,0 +1,35 @@
+[package]
+name = "service-extensions-samples"
+version = "0.0.0"
+
+[dependencies]
+log = "0.4"
+url = "2.4"
+
+# Mandatory for cargo-raze
+[lib]
+path = "fake_lib.rs"
+
+[profile.release]
+lto = true
+opt-level = 3
+codegen-units = 1
+panic = "abort"
+strip = "debuginfo"
+
+
+# To regenerate Cargo Bazel rules based on the above:
+#  $ sudo cargo install cargo-raze
```

Comment:
nit: if you change the instructions to use cargo-raze through bazel, don't forget to update this comment, too!

Reply:
Done.